### ISSUE TITLE
Dark mode

### ISSUE THREAD

**Issue by xplosionmind** [ihucos/counter.dev#47]
Supporting an automatic dark interface for the dashboard

**Comment by ihucos** [comment]
Yeah, would be nice. I'll have to give it a lower priority though. What you mean by automatic. Not sure access to an ambien light sensor in the browser is common. It would have a toggle.

**Comment by xplosionmind** [comment]
By automatic I mean that it is used the CSS property `prefers-color-scheme`
